crypten/mpc/mpc.py

```diff
@@ -1215,9 +1312,6 @@ def set(self, enc_tensor):
 }
```

Comment:
For reference, these are the the "right" versions of `__and__`, `__xor__`, and `__or__`. I don't think it's necessary to implement this now, but just as a reference, these functions replace the operators when something like the following is called:

`y = 1 & x`

In this case Python tries to call `1.__and__(x)`, but `1` doesn't have a method `__and__` for objects of type `MPCTensor`, so it returns `NotImplementedError`. By convention, Python will then try `x.__rand__(1)`.

Reply:
Note: For `__rsub__`, `__radd__`, and `__rmul__`, these are implemented in cryptensor.py here:

https://github.com/MLH-Fellowship/CrypTen/blob/master/crypten/cryptensor.py#L404-L406

Because of this, the "right" versions of these functions don't need to be implemented here and should work automatically.